src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue

```diff
@@ -39,9 +40,20 @@
 			</slot>
 		</span>
 
-		<span v-if="$slots.default" class="checkbox-content__text" :class="[textClass]">
-			<!-- @slot The checkbox/radio label -->
-			<slot />
+		<span class="checkbox-content__wrapper">
+			<span
+				v-if="$slots.default"
+				class="checkbox-content__text"
+				:class="textClass"
+				:aria-describedby="descriptionId">
```

Comment:
This needs to be set on the `<input>`.
Also with a description the `id` set to `aria-labelledby` needs to be the id of this span.

So I would suggest to create the IDs on the parent component (the NcCRS) and pass it to this component as `label-id` and `description-id` then you can set both to the `input` and apply them here to their respective containers.

Reply:
done